dask/dataframe/tests/test_shuffle.py

```diff
@@ -257,8 +258,8 @@ def test_shuffle_sort(shuffle):
     assert_eq(ddf2.loc[2:3], df2.loc[2:3])
 
 
-@pytest.mark.parametrize("shuffle", ["tasks", "disk"])
-@pytest.mark.parametrize("scheduler", ["threads", "processes"])
+@pytest.mark.parametrize("shuffle", ["disk"])
+@pytest.mark.parametrize("scheduler", ["processes"])
```

Comment:
Why did we drop the extra parameters here?

Reply:
Sorry, just debugging.